scripts/restore.sh

```diff
@@ -302,7 +312,7 @@ restore_window_properties() {
 }
 
 restore_shell_history() {
-	awk 'BEGIN { FS="\t"; OFS="\t" } /^pane/ { print $2, $3, $6, $9; }' $(last_resurrect_file) |
+	awk 'BEGIN { FS="\t"; OFS="\t" } /^pane/ { print $2, $3, $6, $10; }' $(last_resurrect_file) |
```

Comment:
I'm trying to get rid of this feature for a long time

https://github.com/tmux-plugins/tmux-resurrect/blob/master/docs/restoring_shell_history.md

Please don't fix this, I'll remove "restore shell history" after we merge restoring pane titles feature.

Reply:
Does it mean I should revert all changes in `restore_shell_history()` as well as `dump_shell_history()`?

Reply:
Yes please.

I'll remove the whole "shell history" feature after we merge this branch.